usr/bin/whonix-gateway-firewall

```diff
@@ -153,6 +200,24 @@ variables_defaults() {
         10.138.0.0/16 \
         10.152.152.10 \
       "
+    else
+      ## Non-Qubes-Whonix:
+      ## IP HARDCODED. If you want to change IP, set variable GATEWAY_IP through a
+      ## drop-in configuration snippet in /etc/whonix_firewall.d
+      ## configuration folder instead.
+      WORKSTATION_DEST_SOCKSIFIED="10.152.152.10"
+    fi
+  fi
+
+  if [ -z "${WORKSTATION_DEST_SOCKSIFIED_IPV6:-}" ]; then
+    ## 10.152.152.10 - Non-Qubes-Whonix-Gateway IP
+    ##
+    ## 10.137.0.0/8  - persistent Qubes-Whonix-Gateway IP range
+    ## 10.138.0.0/8  - DispVM Qubes-Whonix-Gateway IP range
+    if test -f /usr/share/qubes/marker-vm; then
+      ## https://forums.whonix.org/t/whonix-gateway-not-reachable/7484/16
+      ## Qubes-Whonix:
+      ## IP HARDCODED. IP 10.152.152.10 is hardcoded in some places.
       WORKSTATION_DEST_SOCKSIFIED_IPV6="\
```

Comment:
The last IP address set in this variable is still set to `::ffff:10.152.152.10`. Shouldn't it be `fd19:c33d:88bc::10`?

(Github is refusing to let me comment on the specific line that I think may have gotten missed, so I commented on the variable assignment instead.)

Reply:
For non-qubes, yes. But within Qubes-Whonix, it is unlikely to match, unless the workstation doesn't update the IPs to us correctly. It won't hurt to add it though, so I went ahead and also added this IP.